doc/asciidoc/consumer/configuration.adoc

```diff
@@ -21,6 +21,7 @@ kafka.value.deserializer=org.apache.kafka.common.serialization.ByteArrayDeserial
 
 streams.check.apoc.timeout=<ms to await for APOC being loaded, default -1 skip the wait>
 streams.check.apoc.interval=<ms interval awaiting for APOC being loaded, default 1000>
+streams.cluster.only=<boolean value that prevent for starting in single instances usefull when you're performing backup/restore ops, default false>
```

Comment:
Add:

"If set to true, streams will refuse to start in single instance mode, or when run in the context of the backup operation.  This is an important safety guard to ensure that operations do not occur in unexpected situations for production deploys"